_events/sr2023/soton-tech-day-february.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Southampton February Tech Day
+date: 2023-02-04 10:00:00 Europe/London
+end_date: 2023-02-04 17:00:00 Europe/London
+layout: event
+type: tech-day
+location: Southampton University
+---
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They're also an opportunity to see how other
+teams are doing or get more direct help with your robots.
+
+We provide a space for you to work in, with power and internet access, as well
+as volunteers able to help you with your kits and hands-on guidance with your
+robots.
+
+Please [let us know][tech-day-signup] by Friday 27th January 2023 if you'd like to attend.
+
+**Please bring your laptops!**
+
+## Directions
+
+Once on campus, follow the signs to Building 25 and then look for the signage
+for Student Robotics.
+
+* How to [get to campus][soton-campus-directions]
+* [On-campus directions][soton-directions] (and parking)
+* [Where to find food][soton-food-map]. Food is scarce in the Tech Day location,
+  so we recommend either ordering food in or taking packed lunches.
+
+## Schedule
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Doors Open. |
+| 12:30 | There will be a lunch break. Teams should bring their own lunches or get them from food vendors near the venues. |
+| 13:00 | Lightning Talks, where teams will talk about their robots and what they plan on doing with them. |
+| 17:00 | Finish. |
+
+[tech-day-signup]: https://forms.gle/oJ9DK8imzek7fQQb6
+[soton-food-map]: https://goo.gl/yYlfs5
+[soton-directions]: {{ site.baseurl }}/resources/sr2020/southampton-tech-day-directions-highfield-building-25.pdf
+[soton-campus-directions]: http://www.southampton.ac.uk/about/visit/getting-to-our-campuses.page
```

Comment:
Can we make this link TLS? (I had thought that this was checked in CI, though it seems not).

Reply:
> Can we make this link TLS? (I had thought that this was checked in CI, though it seems not).

```suggestion
[soton-campus-directions]: https://www.southampton.ac.uk/about/visit/getting-to-our-campuses.page
```

Reply:
f53f19ccedb7ccb8111538607e2be7075710d447